lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/TitleBarButtonController.java

```diff
@@ -87,6 +87,11 @@ public void sendOnNavigationButtonPressed(String buttonId) {
         getView().sendOnNavigationButtonPressed(buttonId);
     }
 
+    @Override
+    public String getCurrentComponentName() {
```

Comment:
It was complaining that `getCurrentComponetName` needs to be implemented. Should this return its actual componentName?

Reply:
Yes

Reply:
Now returning:

```java
@Override
public String getCurrentComponentName() {
  return button.component.name.get();
}
```